### DISCUSSION TITLE
Minigames for VaGABfS

### DISCUSSION THREAD

**Discussion by Flottegurke** [The-Bug-Bashers/VaGABfS#37] [Ideas]
With the release v2.0 minigames should get introduced, does anyone got ideas for wich minigames should be added?
currently ownly Rock, Paper, sisors is planed.

**Comment by Dinkelbroetchen** [comment]
Blackjack :)
Key functions should be „Hit“ (take a another Card) or „Stand“ wich means, that the dealer draws Cards until he his cards get over a sum of 21 (the dealer loses) or his cards get a higher count than yours (dealer wins).

**Comment by dagadeta** [comment]
You could also implement Chess. Would be a little complicated, but fun.
